core/src/main/java/jenkins/monitor/JavaVersionRecommendationAdminMonitor.java

```diff
@@ -46,6 +46,10 @@
 @Symbol("javaVersionRecommendation")
 public class JavaVersionRecommendationAdminMonitor extends AdministrativeMonitor {
 
+    public JavaVersionRecommendationAdminMonitor() {
+        super(JavaVersionRecommendationAdminMonitor.class.getName() + "2");
```

Comment:
```suggestion
        super(JavaVersionRecommendationAdminMonitor.class.getName() + "-2");
```

Assuming this is legal, to not make people look for a class named `JavaVersionRecommendationAdminMonitor2`?

Reply:
Done